src/highdicom/sr/sop.py

```diff
@@ -770,3 +783,48 @@ def from_dataset(
         sop_instance = super().from_dataset(dataset, copy=copy)
         sop_instance.__class__ = Comprehensive3DSR
         return cast(Comprehensive3DSR, sop_instance)
+
+
+def srread(
+    fp: Union[str, bytes, PathLike, BinaryIO],
+) -> Union[EnhancedSR, ComprehensiveSR, Comprehensive3DSR]:
+    """Read a structured report document in DICOM File Format.
+
+    The object is returned as an instance of the highdicom class corresponding
+    to the dataset's IOD. Currently supported IODs are:
+    - EnhancedSRStorage
+    - ComprehensiveSRStorage
+    - Comprehensive3DSRStorage
```

Comment:
Are you sure the list will render correctly?

Reply:
Good point, resolved in c436b18